alf/environments/suite_carla.py

```diff
@@ -615,6 +626,41 @@ def _get_goal(self):
         return _calculate_relative_position(self._actor.get_transform(),
                                             self._goal_location)
 
+    def update_speed_limit(self, dis_threshold=1.0):
+        """Update the speed limit of the actor according to the active speed
+        limit sign. The speed limit is updated when passing by a speed limit sign.
+
+        Args:
+            dis_threshold (float): the distance within which to consider the
```

Comment:
It the unit meter? I feel 1 meter is too low. It may miss the speed limit sign.

Reply:
The unit is meter. Doc str has been updated. The threshold has been increased to a larger value.